ix-dev/community/urbackup/ix_values.yaml

```diff
@@ -0,0 +1,14 @@
+images:
+  image:
+    repository: uroni/urbackup-server
+    tag: 2.5.x
+
+consts:
+  urbackup_container_name: urbackup-server
+  internal_webui_port: 55414
+  internal_backup_port_1: 55413
+  internal_backup_port_2: 55415
+  internal_backup_port_3: 35623
+  backup_path: /backups
+  database_path: /var/urbackup
+  www_path: /usr/share/urbackup
```

Comment:
what does the wwwfolder contain?

Reply:
<img width="532" alt="image" src="https://github.com/user-attachments/assets/85ae014e-d2e0-478b-a6e7-d8e38fc61094" />

Im not sure why someone would want to bind mount it but i added that option from [here](https://github.com/uroni/urbackup-server-docker/blob/52023db293ddc9a55c00e17b06b49a76e48d11af/README.md?plain=1#L50)

i run this at home and have never mounted that dir. i can remove and maybe people can optionally mount it

Reply:
Yes let's remove it. It's probably for modifying the webui.